src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/workflow/WorkflowManager.kt

```diff
@@ -18,14 +19,37 @@ class WorkflowManager(sync: InstanceSync) : InstanceService(sync) {
         aem.prop.boolean("instance.workflowManager.restoreIntended")?.let { set(it) }
     }
 
+    fun schedule(modelId: String, node: Node) = workflow(modelId).schedule(node)
+
     fun workflow(id: String) = Workflow(this, id)
 
+    fun workflow(id: String, options: Workflow.() -> Unit) = Workflow(this, id).run(options)
+
     fun workflows(type: String) = workflows(listOf(type))
 
     fun workflows(types: Iterable<String>) = WorkflowType.ids(types).map { Workflow(this, it) }
 
     fun workflows(vararg types: String) = workflows(types.asIterable())
 
+    fun queryNodes(path: String, type: String): Sequence<Node> = instance.sync.repository {
```

Comment:
why it is public...

WorkflowManager.queryNodes

from the public point of view... what does it mean? ... :/

Reply:
it should be in scheduler.

Reply:
and the name may be better...

findPayloadResources()
or findPayloads()

in workflow nomenclatures the payload is a resource for which workflow is running